pkg/scanner/scan.go

```diff
@@ -121,6 +122,12 @@ func (s Scanner) ScanArtifact(ctx context.Context, options types.ScanOptions) (t
 		log.Logger.Warnf("The vulnerability detection may be insufficient because security updates are not provided")
 	}
 
+	if artifactInfo.Type == ftypes.ArtifactFilesystem {
```

Comment:
I want to define artifact.Artifact.Clean(types.ArtifactReference) and call it in defer.

Reply:
done